core/store/orm/config.go

```diff
@@ -202,6 +204,33 @@ func init() {
 		OptimismGasFees:                  true,
 	}
 
+	// Fantom
+	fantomTestnet := ChainSpecificDefaultSet{
```

Comment:
Probably its safer to do something like this instead:

```
fantomTestnet := mainnet
fantomTestnet.EthGasLimitDefault = 50000
etc...
```

We should probably take this approach for all the chains because I've seen bugs from accidentally missing vars slip through before